### PR TITLE
Some cleanup, documentation and added mongo db name to config

### PR DESCRIPTION
**Pull request by Areedtz**

### CODE REVIEW

File: src/classification/segmented_audio_analysis.py [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 
-from utilities.get_song_id import get_song_id
-from utilities.filehandler.handle_path import get_absolute_path
-from utilities.filehandler.handle_audio import get_MonoLoaded_Song
-from similarity.split_song import split_song
-from classification.classifier.profile_data_extractor import get_classifier_data
-from classification.extractor.low_level_data_extractor import make_low_level_data_file
 from bpm.bpm_extractor import get_song_bpm
+from classification.classifier.profile_data_extractor import \
+    get_classifier_data
```
Comment (QuincySharp) [inline src/classification/segmented_audio_analysis.py]: Come on...

File: src/classification/segmented_audio_analysis.py [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+from classification.extractor.low_level_data_extractor import \
+    make_low_level_data_file
+from essentia.standard import MonoWriter
```
Comment (QuincySharp) [inline src/classification/segmented_audio_analysis.py]: 3rd party should be above local modules no? pep8?

File: src/classification/api_helper.py [hunk 3]
```diff
@@ -2,17 +2,21 @@
 from tempfile import NamedTemporaryFile
 
 from bpm.bpm_extractor import get_song_bpm
-from classification.extractor.low_level_data_extractor import make_low_level_data_file
-from classification.classifier.profile_data_extractor import get_classifier_data
+from classification.classifier.profile_data_extractor import \
+    get_classifier_data
+from classification.extractor.low_level_data_extractor import \
+    make_low_level_data_file
```
Comment (QuincySharp) [inline src/classification/api_helper.py]: Yikes

File: src/classification/segmented_audio_analysis.py (hunk 1 above)
Comment (Areedtz) [reply src/classification/segmented_audio_analysis.py]: Pep 8 length of lines ;)

File: src/classification/segmented_audio_analysis.py (hunk 2 above)
Comment (Areedtz) [reply src/classification/segmented_audio_analysis.py]: Nicely spotted, fixed :)

File: src/classification/api_helper.py (hunk 3 above)
Comment (Areedtz) [reply src/classification/api_helper.py]: Pep 8 max line length :)

**Pull request merged**